### PR TITLE
epstool: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by maxim-belkin**

### CODE REVIEW

File: Formula/epstool.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Epstool < Formula
   end
 
   depends_on "ghostscript"
+  depends_on "gcc" => :build unless OS.mac?
```
Comment (sjackman) [inline Formula/epstool.rb]: Better to use either `needs :cxx11` or `needs :cxx14` or `needs :openmp` if that's the reason for needing a modern compiler, or failing that
```ruby
fails_with :gcc => "4.8"
```
There's a bug currently that causes it to fall back to `/usr/bin/cc`, which may also be too old.
The workaround for that bug is as you have above, but it's just a hack work around.

Comment (maxim-belkin) [inline Formula/epstool.rb]: I don't think that we need `cxx11`, `cxx14`, or `openmp` to compile this. We just need a header file with correct call for `O_CREAT`

Comment (sjackman) [inline Formula/epstool.rb]: I fixed the bug in `brew` (falling back to `/usr/bin/cc`), so the following should work:
```diff
-depends_on "gcc" => :build unless OS.mac?
+fails_with :gcc => "4.8"
```

Comment (maxim-belkin) [inline Formula/epstool.rb]: do I need to rebase it?

Comment (sjackman) [inline Formula/epstool.rb]: Always a good idea to rebase, but it shouldn't actually matter.

**Comment by maxim-belkin** [comment]
Circle failed with an error message:
```
mkdir ./bin
/home/linuxbrew/.linuxbrew/bin/gcc-5 -I./src -I./epsobj -Os -w -pipe -march=core2 -DX11 -DUNIX -DNONAG  -Wall -Wstrict-prototypes -Wmissing-declarations -Wmissing-prototypes -fno-builtin -fno-common -Wcast-qual -Wwrite-strings  -g      -o ./epsobj/xnodll.o -c ./src/xnodll.c
mkdir ./epsobj
Assembler messages:
Fatal error: can't create ./epsobj/xnodll.o: No such file or directory
echo "dummy" > ./epsobj/lib.rsp
```

**Comment by sjackman** [comment]
`ENV.deparallelize` ?

**Comment by sjackman** [comment]
@maxim-belkin @rwhogg What was the error message that this `fails_with` fixed?
```ruby
fails_with :gcc => "4.8"
```

**Pull request closed without merging**